### ISSUE TITLE
Dev/Build thread panic error introduced with v0.6.2

### ISSUE THREAD

**Issue by nicolo-tito** [blitz-js/next-superjson-plugin#101]
### Verify Next.js canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Describe the bug

Upgrading from 0.6.1 to 0.6.2 introduced this bug that prevent execution both during next dev and next build:

thread '<unnamed>' panicked at C:\Users\Administrator\.cargo\registry\src\index.crates.io-6f17d22bba15001f\rkyv-0.7.42\src\impls\core\mod.rs:267:17:
assertion failed: !result.is_null()

### Expected behavior

Dev and Build running without error

### Version

0.6.2

### Config

```javascript
const nextConfig = {
  output: 'standalone',
  experimental: {
    swcPlugins: [['next-superjson-plugin', {}]]
  }
}
```

### Additional context

```
Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 11 Pro
Binaries:
  Node: 20.11.0
  npm: N/A
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 14.1.0
  eslint-config-next: 14.1.0
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.3.3
Next.js Config:
  output: standalone
```

**Comment by nicolo-tito** [comment]
Reverting to 0.6.1 fix the error

**Comment by orionmiz** [comment]
Check your Next.js version again. It's not canary release.

**Comment by nicolo-tito** [comment]
> Check your Next.js version again. It's not canary release.

With latest canary I got 

thread '<unnamed>' panicked at C:\Users\Administrator\.cargo\registry\src\index.crates.io-6f17d22bba15001f\wasmer-wasix-0.18.0\src\runtime\task_manager\tokio.rs:105:19:
there is no reactor running, must be called from the context of a Tokio 1.x runtime

Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 11 Pro
Binaries:
  Node: 20.11.0
  npm: N/A
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 14.1.1-canary.52 // Latest available version is detected (14.1.1-canary.52).
  eslint-config-next: 14.1.0
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.3.3
Next.js Config:
  output: standalone
